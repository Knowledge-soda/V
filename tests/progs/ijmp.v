print(ijmp(3, place) print(8) place: 12)
print(ijmp(0, place2) print(8) place2: 12)
