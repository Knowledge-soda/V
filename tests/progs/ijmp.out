12
8
12
